pages/Types.html

```diff
@@ -657,6 +658,8 @@ <h2 id="XMLHttpRequest"> XMLHttpRequest </h2>
 <h2 id="jqXHR"> jqXHR </h2>
 <p>As of jQuery 1.5, the <a href="/jQuery.ajax/">$.ajax()</a> method returns the jqXHR object, which is a superset of the XMLHTTPRequest object. For more information, see the <a href="/jQuery.ajax/#jqXHR">jqXHR section of the $.ajax entry</a>
 </p>
+<h2 id="Thenable">Thenable</h2>
+<p>Any object that has a <code>then</code> method.</p>
```

Comment:
@mgol I considered that but found it more confusing than useful. All functions are objects. So this isn't contradictory.